network/links.md

```diff
@@ -77,28 +85,36 @@ default the `-p` flag binds the specified port to all interfaces on
 the host machine. But you can also specify a binding to a specific
 interface, for example only to the `localhost`.
 
+```bash
     $ docker run -d -p 127.0.0.1:80:5000 training/webapp python app.py
+```
```

Comment:
Same here ("console" and "un-indent")

```suggestion
```console
$ docker run -d -p 127.0.0.1:80:5000 training/webapp python app.py
```
```